.setup/data/courses/development.yml

```diff
@@ -289,6 +289,30 @@ gradeables:
     eg_has_release_date: true
     eg_max_random_submissions: 0
 
+  - gradeable_config: cpp_updated_random_input_output
```

Comment:
rather than adding a new gradeable, let's just modify the existing cpp_random_input_output example to replace the python solution with a cpp solution

Reply:
Alright, got it
Will modify the existing example itself.